code/src/components/EmptyList.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+
+import './EmptyList.css'
+
+export const EmptyList = () => {
+  return (
+    <div className='gif-wrapper'>
+      <img 
+      className='empty-list-gif'
+      src={require('../media/animation.gif')}
+      alt='GIF of a seal with sun glasses'
+      />
+      <h2>There is nothing left to do!</h2>
+      <p className='sub-title'>Take a break or add another to-do!</p>
+    </div>
+  );
+};
```

Comment:
Here you are using semicolons, choose whether to use them or not and be consistent